<launch>

	<node  name="detection_tensorflow" pkg="era_gazebo" type="detection_tensorflow.py" output="screen">

		<remap from="image_input" to="/camera/rgb/image_raw" />

	</node>


</launch>